bitnami/mariadb-galera/Chart.yaml

```diff
@@ -28,4 +28,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-mariadb-galera
   - https://github.com/prometheus/mysqld_exporter
   - https://mariadb.org
-version: 5.2.1
+version: 5.2.2
```

Comment:
Since this is a feature I would bump to 5.3.0.

Reply:
Also done :)